stars/default/name.fab

```diff
@@ -1360,6 +1502,7 @@
  62886|35_Com
  63355|36_Com
  63462|37_Com
+ 63462|13_CVn*
```

Comment:
Why asterisk added here?

Reply:
```suggestion
 63462|13_CVn*
```
According to [37_Com](https://en.wikipedia.org/wiki/37_Comae_Berenices) and [Table of stars with Flamsteed designations](https://en.wikipedia.org/wiki/Table_of_stars_with_Flamsteed_designations)

Reply:
This star hasn’t asterisk in designation